vim/settings/neocomplete.vim

```diff
@@ -1,34 +0,0 @@
-" neocomplete
-" Next generation completion framework.
-
-let g:acp_enableAtStartup = 0
-let g:neocomplete#enable_at_startup = 1
-let g:neocomplete#enable_camel_case = 1
-let g:neocomplete#enable_smart_case = 1
```

Comment:
Are all configurations in this file present in deoplete? I don't see them explicit set in your PR and we don't want to introduce regressions in behavior for the users of this repo

Reply:
Again the details are here 
https://github.com/Shougo/deoplete.nvim

Reply:
Sorry, that doesn't answer my question. You're removing configurations that might effect all users of this dotfiles repo. Pointing to deoplete's repo doesn't answer my question and the inherent risk of removing those configs. There's also another question above pending an answer :)

Thank you

Reply:
Looks like a more complete `deoplete.vim` would look like:

```
let g:deoplete#enable_at_startup = 1

call deoplete#custom#option({
\ 'camel_case': v:true,
\ 'smart_case': v:true,
\ })
```

There does not appear to be a `let g:acp_enableAtStartup = 0` setting for `deoplete`.

Not sure if this is correct but it is what I am using with this PR.